pages/api/getUser.js

```diff
@@ -0,0 +1,32 @@
+import nc from 'next-connect'
+import prisma from '../../lib/adapters/prismaClient'
+
+const handler = nc()
+  .get(async(req, res) => {
+    const data = req.query
+
+    const users = await prisma.users.findUnique({
+      where : {
+        id : parseInt(data.user_id)
+      },
+
+      include : {
+        posts : {
+          select : {
+            author : true,
+            content : true,
+            post_id : true,
+            user_id : true,
+            image_url : true,
+          }
+        }
+      },
+    })
+
+    return res.json({
+      users
+    })
```

Comment:
Similar comment here, can't we pass the user object directly? 
Then we can remove inside `[uid].js` line 12 `const userData = fetchedUserData.users`, since we're passing the object directly 🤔 

```suggestion
    return res.json(users)
```